scalability_and_performance/index.adoc

```diff
@@ -0,0 +1,61 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index"]
+= {product-title} scalability and performance overview
+include::_attributes/common-attributes.adoc[]
+:context: index
+
+// ifndef::openshift-dedicated,openshift-rosa[]
+{product-title} offers best practices and utilities to help you optimize the performance and scale of your {product-title} clusters. The following documentation provides information on recommended performance and scalability practices, reference design specifications, optimization and low latency tuning.
+
+To engage with Red Hat support, see xref:../support/getting-support.adoc#getting-support[Getting support].
+
+// endif::openshift-dedicated,openshift-rosa[]
+
+[NOTE]
+====
+Some performance and scalability Operators have release cycles that are independent from {product-title} release cycles. For more information see link:access.redhat.com/support/policy/updates/openshift_operators[Openshift Operators].
+====
+
+[discrete]
+==== Recommended performance and scalability practices
```

Comment:
This renders as H4. a11y-wise, it should be an H2. Same with other headings past the H1, if I'm guessing right.

Reply:
Done, thank you!